### PR TITLE
Add changelog entries from PR #100

### PR DESCRIPTION
**Pull request by sargas**
PR to note the changes that #97 would have done, and warn about changing science results.

### CODE REVIEW

**Comment by cdeil** [comment]
👍

**Comment by sargas** [comment]
I managed to change the API in another way that's undocumented: `WCS` objects are not sufficient for conversion to image coordinates.

This is because angles are defined based on the rotation of the north axis onto the pixel plane, as measured by the *center of the FITS image*. It would make more sense to me to make it at the center of the shape in question, or even based on CRPIX[12], but this is how DS9 does it. Added the documentation update to this PR.

**Comment by cdeil** [comment]
@sargas - Why wasn't this merged? Is it ready to go? (if yes, just go ahead and merge, please)

File: docs/getting_started.rst [hunk 1]
```diff
@@ -135,16 +135,9 @@ converted to image coordinates. ::
...
-This is useful if your FITS file is not a simple 2D image, as you can then use only the celestial
-subset of the co-ordinates to parse the region: ::
-
-    from astropy.io import fits
-    from astropy.WCS import WCS
-    f = fits.open("t1.fits")
-    w = WCS(f[0].header)
-    w_im = w.celestial
-    r2 = pyregion.parse(region_string).as_imagecoord(w_im)
+`ShapeList.as_imagecoord <pyregion.ShapeList.as_imagecoord>` will use the
```
Comment (bsipocz) [inline docs/getting_started.rst]: I'm not sure this will be properly rendered, but suggest to merge now and follow-up with a fix if needed.

**Comment by bsipocz** [comment]
I take the blame, and push the button now...

**Comment by cdeil** [comment]
@sargas  and @bsipocz - Thanks!

**Pull request merged**